extensions-core/parquet-extensions/src/main/java/org/apache/druid/data/input/parquet/simple/ParquetGroupConverter.java

```diff
@@ -52,10 +52,14 @@ public class ParquetGroupConverter
   private static final long NANOS_PER_MILLISECOND = TimeUnit.MILLISECONDS.toNanos(1);
 
   /**
-   * See {@link ParquetGroupConverter#convertField(Group, String)}
+   * Convert a parquet group field as though it were a map. Logical types of 'list' and 'map' will be transformed
+   * into java lists and maps respectively ({@link ParquetGroupConverter#convertLogicalList} and
+   * {@link ParquetGroupConverter#convertLogicalMap}), repeated fields will also be translated to lists, and
+   * primitive types will be extracted into an ingestion friendly state (e.g. 'int' and 'long'). Finally,
+   * if a field is not present, this method will return null.
    */
   @Nullable
-  private static Object convertField(Group g, String fieldName, boolean binaryAsString)
```

Comment:
Just double checking, this `binaryAsString` argument.  That was being passed around but never actually used?

Any changes you want to make to `ParquetToJson` are good, that code is just for our own dev purposes, but these changes in `ParquetGroupConverter` is changing the code that our current parquet-based file ingestions end up using.  So, I just want to double check that this cleanup is truly a redundant argument and not just something that wasn't needed for `ParquetToJson` but used for the other production code paths.

Reply:
`binaryAsString` was passed around in `private static` methods in a class on which an `instance method` was called first....so I've choosen to remove them (and use the implict class access - to get it where needed) ; instead of adding another boolean to every static method